packages/asset-swapper/src/utils/market_operation_utils/path_optimizer.ts

```diff
@@ -11,13 +11,16 @@ import { MarketOperation, NativeOrderWithFillableAmounts } from '../../types';
 import { VIP_ERC20_BRIDGE_SOURCES_BY_CHAIN_ID, ZERO_AMOUNT } from './constants';
 import { dexSamplesToFills, ethToOutputAmount, nativeOrdersToFills } from './fills';
 import { DEFAULT_PATH_PENALTY_OPTS, Path, PathPenaltyOpts } from './path';
+import { TestDataSampler } from './router_test_data_sampler';
 import { DexSample, ERC20BridgeSource, FeeSchedule, Fill, FillData, SamplerMetrics } from './types';
 
 // tslint:disable: prefer-for-of custom-no-magic-numbers completed-docs no-bitwise
 
 const RUN_LIMIT_DECAY_FACTOR = 0.5;
 // NOTE: The Rust router will panic with less than 3 samples
 const MIN_NUM_SAMPLE_INPUTS = 3;
+const SAMPLE_THRESHOLD = 0.003;
```

Comment:
NIT: maybe we can export this from `router_test_data_sampler` so we have all configuration in one file

Reply:
Good idea